sh.exe.stackdump

```diff
@@ -0,0 +1,22 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF8D7F2AAB7
```

Comment:
Your tests are passing. However, the `sh.exe.stackdump` file is still present. May you delete it and then run `contribute`, option `[3]`? Thanks @kostromins.